timemachine/cpp/src/gpu_utils.cuh

```diff
@@ -36,6 +38,16 @@ inline void curandAssert(curandStatus_t code, const char *file, int line, bool a
    }
 }
 
+#define NVRTC_SAFE_CALL(x)                                        \
+  do {                                                            \
+    nvrtcResult result = x;                                       \
+    if (result != NVRTC_SUCCESS) {                                \
+      std::cerr << "\nerror: " #x " failed with error "           \
+                << nvrtcGetErrorString(result) << '\n';           \
+      exit(1);                                                    \
+    }                                                             \
+  } while(0)
```

Comment:
Is this do while required?

Reply:
It's not - but this macro isn't used, so I will remove it.